run_smalltextpad.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
Do you actually need it? You control the environment so it makes no sense to run these checks (and export DISPLAY),